src/langs/en/library.json

```
@@ -74,5 +74,24 @@
   "SIGN_IN_MODAL_CLOSE_BUTTON_ARIA_LABEL": "Close Sign In Modal",
   "VIEW_BUTTON_TOOLTIP": "View item",
   "SHARED_ITEMS_LABEL": "Shared Items",
-  "OWN_ITEMS_LABEL": "Owned Items"
+  "OWN_ITEMS_LABEL": "Owned Items",
+  "SUMMARY_DETAILS_TITLE": "Details",
+  "SUMMARY_DETAILS_CREATED_AT_TITLE": "Created",
+  "SUMMARY_DETAILS_UPDATED_AT_TITLE": "Updated",
+  "SUMMARY_DETAILS_NO_LANGUAGES": "This item has no defined language.",
+  "SUMMARY_DETAILS_NO_CATEGORIES": "This item has no categories.",
+  "SUMMARY_DETAILS_LICENSE_TITLE": "License",
+  "SUMMARY_BROWSE_FILE_UPDATED": "Updated {{date}}",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_zero": "Empty Folder",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_one": "{{count}} File",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_other": "{{count}} Files",
+  "SUMMARY_DESCRIPTION_SHOW_MORE": "Show More",
+  "SUMMARY_DESCRIPTION_SHOW_LESS": "Show Less",
+  "SUMMARY_ITEMS_SHOW_LESS_one": "- Hide {{count}} item",
+  "SUMMARY_ITEMS_SHOW_LESS_other": "- Hide {{count}} items",
+  "SUMMARY_ITEMS_SHOW_MORE": "+ View {{count}} more",
+  "SUMMARY_ACTIONS_PREVIEW_CONTENT": "Preview Content",
```

Maybe simply use 
```suggestion
  "SUMMARY_ACTIONS_PREVIEW_CONTENT": "Preview",
```
as you already have the "play" button.